src/test/debugger/appWorker.test.ts

```diff
@@ -0,0 +1,206 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+
+import * as assert from "assert";
+import * as WebSocket from "ws";
+import * as path from "path";
+import * as Q from "q";
+import * as sinon from "sinon";
+
+import * as AppWorker from "../../debugger/appWorker";
+import {ScriptImporter} from "../../debugger/scriptImporter";
+
+suite("appWorker", function() {
+    suite("debuggerContext", function() {
+        suite("SandboxedAppWorker", function() {
+
+            const sourcesStoragePath = path.resolve(__dirname, "assets");
+            const startScriptFileName = require.resolve(path.join(sourcesStoragePath, ScriptImporter.DEBUGGER_WORKER_FILE_BASENAME));
+            const debugAdapterPort = 9090;
+
+            let sandboxedWorker: AppWorker.SandboxedAppWorker;
+            let downloadAppScriptStub = sinon.stub();
+            let postReplyFunction = sinon.stub();
+            let readFileStub = sinon.stub();

```

Comment:
We need the declaration outside the `suiteSetup` so it is visible within the tests. In this case the assignments could be delayed to the `setup`, if we wanted to separate definition/declaration.